src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityHelper.java

```diff
@@ -165,19 +165,19 @@
     }
     
     /**
-     * Gets the {@link DockerEventsAction} of Jenkins instance.
+     * Gets the {@link DockerTraceabilityRootAction} of Jenkins instance.
      * @return Instance or null if it is not available
      */
-    public static @CheckForNull DockerEventsAction getTraceabilityAction() {
+    public static @CheckForNull DockerTraceabilityRootAction getTraceabilityAction() {
```

Comment:
I'll rename it to <code>getInstance()</code> and move to the <code> DockerTraceabilityRootAction</code> class